src/modules/auth/store/action.js

```diff
@@ -1,24 +1,23 @@
 /* eslint-disable max-lines */
-import { to } from 'await-to-js';
-import { toast } from 'react-toastify';
-import loginTypes from 'src/modules/auth/const/loginTypes';
-import { tokenMap } from '@token/fungible/consts/tokens';
-import { extractAddress as extractBitcoinAddress } from '@wallet/utils/api';
-import { getConnectionErrorMessage } from '@network/utils/getNetwork';
-import { extractKeyPair } from '@wallet/utils/account';
-import { getAccounts } from '@wallet/store/action';
-import { defaultDerivationPath } from 'src/utils/explicitBipKeyDerivation';
+// import { to } from 'await-to-js';
+// import { toast } from 'react-toastify';
+// import loginTypes from 'src/modules/auth/const/loginTypes';
+// import { tokenMap } from '@token/fungible/consts/tokens';
+// import { extractAddress as extractBitcoinAddress } from '@wallet/utils/api';
```

Comment:
bitcoin and token map is out of the scope pleaser revert other wise it will get conflict with the other pr